dataactvalidator/filestreaming/schemaLoader.py

```diff
@@ -86,10 +91,10 @@ def loadCrossRules(filename):
                     targetFileId = validationDb.getFileId(record["target_file"])
                 else:
                     targetFileId = None
-                validationDb.addMultiFieldRule(
+                validationDb.addRule(
                     fileId, record["rule_type"], record["rule_text_one"],
-                    record["rule_text_two"], record["description"],
-                    record["rule_label"], record["rule_timing"], targetFileId)
+                    record["rule_text_two"], record["description"], record["rule_timing"],
+                    record["rule_label"], targetFileId, fileId = fileId)
```

Comment:
They do not currently always have a target file as I implemented the first rule types before you added the target file field.  Refactoring those to use target file would be worthwhile, it's on my list of refactors to do if I have time to spare at the end of this sprint.